boards/xmc-2go/include/periph_conf.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2015 Sebastian Sontberg <sebastian@sontberg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_xmc2go
+ * @{
+ *
+ * @file
+ * @brief       Peripheral MCU configuration for the xmc-2go board
+ *
+ * @author      Sebastian Sontberg <sebastian@sontberg.de>
+ */
+
+#ifndef PERIPH_CONF_H_
+#define PERIPH_CONF_H_
+
+#include "cpu.h"
+#include "periph_cpu.h"
+
+#include "periph/usic_asc.h"
+#include "periph/usic_ssc.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name RTC configuration
+ * @{
+ */
+#define RTC_NUMOF          (1U)
+#define RTC_IRQ_PRIO       CPU_DEFAULT_IRQ_PRIO
+#define RTC_DIVIDER        (0x7fff)
+/** @} */
+
+/**
+ * @name PRNG configuration
+ * @{
+ */
+#define RANDOM_NUMOF       (1U)
+/** @} */
+
+/**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF          (2U)
+#define ADC_0_EN           1
+#define ADC_1_EN           0
+#define ADC_MAX_CHANNELS   8
+/** @} */
+
+/**
+ * @name Timer configuration
+ * @{
+ */
+#define TIMER_NUMOF        (1)
+#define TIMER_0_EN         (1)
+#define TIMER_0_CHANNELS   (2)
+#define TIMER_0_MAX_VALUE  (0xffffffff)
+#define TIMER_IRQ_PRIO     CPU_DEFAULT_IRQ_PRIO
+
+#define XTIMER_BACKOFF     (40)
+#define XTIMER_ISR_BACKOFF (30)
+/** @} */
+
+/**
+ * @name UART configuration
+ * @{
+ */
+#define UART_NUMOF         (1U)
+#define UART_0_EN          (1)
+#define UART_IRQ_PRIO      CPU_DEFAULT_IRQ_PRIO
+
+static const uart_instance_t uart_instance[] = {
+    {
+        .usic   = USIC0_CH0,
+        .mode   = &_xmc_usic_asc_controls,
+        .tx_pin = GPIO_ALT(P2, 1, GPIO_ALT_OUT_6),
+        .rx_pin = GPIO_ALT(P2, 2, 6)
+    }
+};
+/** @} */
+
+/**
+ * @name SPI configuration
+ * @{
+ */
+#define SPI_NUMOF          (1)
+#define SPI_0_EN           (1)
```

Comment:
For SPI we can't use the (sizeof ... / sizeof ...) trick because of the expansion of SPI_NUMOF in `drivers/include/periph/spi.h:37:5`. correct?